src/checkout-buttons/strategies/masterpass/masterpass-button-strategy.ts

```diff
@@ -28,19 +29,15 @@ export default class MasterpassButtonStrategy implements CheckoutButtonStrategy
         return this._store.dispatch(this._checkoutActionCreator.loadDefaultCheckout())
             .then(state => {
                 const paymentMethod = state.paymentMethods.getPaymentMethod(methodId);
-                const storeConfig = state.config.getStoreConfig();
+                const locale = formatLocale(this._locale);
```

Comment:
In `masterpass-customer-strategy.ts` there is no variable created for locale, can we please implement it in either way?